browser/algorithms.json

```diff
@@ -50,24 +50,24 @@
     "id": ""
   },
   "sha256": {
-    "sign": "ecdsa",
+    "sign": "ecdsa/rsa",
     "hash": "sha256",
-    "id": ""
+    "id": "3031300d060960864801650304020105000420"
```

Comment:
Added these to get RSA going, which is the only thing it impacts (as I'm sure you know). Although, it is a little weird to do

Reply:
yeah this file was just set up before I latter learned how weird some of the names were handled in node